curriculum/challenges/english/25-front-end-development/lecture-styling-lists-and-links/672b952d3a5c603908841971.md

```diff
@@ -2,7 +2,7 @@
 id: 672b952d3a5c603908841971
 title: Why Are Default Link Styles Important for Usability on the Web?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: 9ta8m4T9KYc
```

Comment:
This is the video id for `What is the CSS filter property`

Reply:
I just messaged Beau about updating the link with the correct one. He is just coming back from a conference so he will probably respond back within a day or so.

Reply:
Beau updated the sheet.
Here is the correct link

https://www.youtube.com/watch?v=Jp3Qd4LU93E